### ISSUE TITLE
CalendarView: Tooltip is not provided for "Next" and "Previous" month buttons.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3525]
**Describe the bug**

**Steps to reproduce the bug**

1. Open Xaml Controls Gallery and navigation to CalendarView page
2. Navigate to Previous, Next buttons
3. Observe the issue.

**Expected behavior**

Tooltip should be provided for "Next" and "Previous" month buttons.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97656599-0bdbd300-1a25-11eb-879a-d3499e4c345c.png)

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/20808340)

**Comment by mdtauk** [comment]
As you have access to the month names, should they not be in the tooltip/accessibility name?

```
                        (Previous: February)     (Next: April)
March                           🔺                    🔻
```

**Comment by StephenLPeters** [comment]
I think that would be awesome, but probably requires winui 3. we can get static labels in winui2

**Comment by Felix-Dev** [comment]
Binding the button tooltip content to the UIA name property of the respective button will give us "Previous" and "Next" as tooltips. Seems fine for now and we can add the month name once WinUI 3 has been open-sourced (for which we should create a new tracking item/proposal).

I would like to work on this.
